airflow/contrib/operators/kubernetes_operator.py

```diff
@@ -137,9 +143,15 @@ def from_job_yaml(job_yaml_string,
             cs['env'] = env
             container_specs.append(cs)
 
+        if 'volumes' in job_data['spec']['template']['spec']:
```

Comment:
This could be replaced with `volumes = job_data['spec']['template']['spec'].get('volumes', [])` but whatever you think is clearer

Reply:
done